service/matchers_test.go

```diff
@@ -0,0 +1,14 @@
+package service
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSvc(t *testing.T) {
+	assert := assert.New(t)
+
+	_, err := NewOsSvc("servicetst", "systemd")
```

Comment:
You are testing this in some other places, I don't see the meaning of this test.